stream-chat-android-ui-components-sample/build.gradle

```diff
@@ -149,4 +149,6 @@ dependencies {
     implementation Dependencies.coil
     implementation Dependencies.coilGif
     implementation Dependencies.shimmer
+
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.6'
```

Comment:
Do we want to add this properly, or was it just a temporary local thing?

Reply:
ooops, sorry